pyo3-bytes/src/bytes.rs

```diff
@@ -93,6 +93,7 @@ impl PyBytes {
     // By setting the argument to PyBytes, this means that any buffer-protocol object is supported
     // here, since it will use the FromPyObject impl.
     #[new]
+    #[pyo3(signature = (buf = PyBytes(Bytes::new())))]
```

Comment:
Continuing from https://github.com/developmentseed/obstore/pull/157#discussion_r1932733891

You're right it doesn't go through to the python text signature, but that means it's currently empty.

<img width="350" alt="image" src="https://github.com/user-attachments/assets/9522a89b-d277-44f8-869f-d104129cbc71" />

We should probably include `text_signature` as well, with `buf` set to `b""`

Reply:
I don't follow.... the signature does go through as `(buf)` -- you are saying you want to explicitly set it to `(buf = b"")`?


```
In [12]: inspect.signature(ry.Bytes)
Out[12]: <Signature (buf)>
```

Reply:
Yes, setting the text_signature to `b""` makes it clear that the default value of `buf` is an empty byte string.

Reply:
<img width="245" alt="image" src="https://github.com/user-attachments/assets/8820bcc9-16a7-4b54-8119-c845c059274f" />

So the `text_signature` is valid input